spec/betterc.dd

```diff
@@ -0,0 +1,47 @@
+Ddoc
+
+$(SPEC_S Better C,
+
+`-betterC` is a Command-Line flag for `dmd`, which generates light-weight,
+Druntime-free binaries with are suitable for linking from C.
+
+---
+extern(C) void main()
+{
+    import core.stdc.stdio : printf;
+    printf("Hello betterC\n");
+}
+---
+
+$(CONSOLE
+> dmd -betterC hello.d && ./hello
+Hello betterC
+)
+
+$(H2 $(LNAME2 consequences, Consequences))
+
+As no DRuntime is available, many D features won't work.
+For example:
+
+$(OL
+    $(LI Garbage Collection)
+    $(LI Thread-local storage)
+    $(LI TypeInfo and ModuleInfo)
+    $(LI `extern(D)` Classes)
```

Comment:
My [investigation](https://github.com/dlang/dlang.org/pull/1796#discussion_r125881603) showed that indeed currently classes are not usable (at least without hacks) without linking druntime, so unfortunately you need to remove this line.